PIPELINE_CHANGELOG.md

```diff
@@ -0,0 +1,96 @@
+<a name="2018-12-14"></a>
+### 2018-12-14
+
+
+#### Features
+
+* **status**
+  * prepare task for website update	 ([aea9fcf](/../../commit/aea9fcf))
```

Comment:
these commit hashes should probably not be shown on the site, since they link to nothing

Reply:
👍